### DISCUSSION TITLE
谢谢大佬，虽然现在直接看也不卡，但我还是用自己的token观看~

### DISCUSSION THREAD

**Discussion by zzpop** [Bssn520/AnimationAutoUpdate#8] [General]
大家在data\ 目录下新建一个mytoken.txt，把自己的token填进去就可以了。
分享两个token获取方法：
方法一：在登录[阿里云盘](https://www.aliyundrive.com/drive/)后，在控制台粘贴 `JSON.parse(localStorage.token).refresh_token`
方法二：用[官方的API](https://alist.nn.ci/zh/guide/drivers/aliyundrive.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C)，用阿里云盘APP扫码获取

**Comment by Bssn520** [comment]
不太明白你的意思，你大概是xiaoya alist的用户吧？alist官方版本暂时没有多token的方式。
之前也想过每个人填写自己的token进行观看，不过，就目前来看，直接挂载阿里云盘分享的方式，应该活不久了......只好作罢

**Reply by zzpop** [reply]
啊？只有xiaoya的才可以这样吗？我以为是alist通用的呢，那当我没说吧，谢谢了

**Reply by zzpop** [reply]
555哭了，斗罗大陆4K没得看了

**Reply by zzpop** [reply]
改用[阿里云盘Open](https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C) 方式挂载满速了

**Reply by Bssn520** [reply]
> 改用[阿里云盘Open](https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C) 方式挂载满速了

以后最高大概只能看1080P了👀

**Reply by zzpop** [reply]
> > 改用[阿里云盘Open](https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C) 方式挂载满速了
> 
> 以后最高大概只能看1080P了👀

感谢，有的看就很不错啦

**Reply by Bssn520** [reply]
> > > 改用[阿里云盘Open](https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C) 方式挂载满速了
> > 
> > 
> > 以后最高大概只能看1080P了👀
> 
> 感谢，有的看就很不错啦

webdav能看原画
